rollup.config.js

```diff
@@ -0,0 +1,52 @@
+import { terser } from "rollup-plugin-terser";
+import ts from "@wessberg/rollup-plugin-ts";
+import resolve from "@rollup/plugin-node-resolve";
+import babel from "@rollup/plugin-babel";
+import pkg from "./package.json";
+import license from "rollup-plugin-license";
+import filesize from "rollup-plugin-filesize";
+
+const extensions = [".js", ".ts"];
+
+export default [
+  {
+    input: "src/HanziWriter.ts",
+    output: [
+      {
+        file: pkg.main,
+        format: "cjs",
```

Comment:
why not umd instead of cjs? Then it could be used is both browsers and in node, no?